src/transformers/models/blenderbot/configuration_blenderbot.py

```diff
@@ -160,6 +161,7 @@ def __init__(
         self.num_hidden_layers = encoder_layers
         self.gradient_checkpointing = gradient_checkpointing
         self.scale_embedding = scale_embedding  # scale factor will be sqrt(d_model) if True
+        self.encoder_no_repeat_ngram_size = encoder_no_repeat_ngram_size
```

Comment:
since `encoder_no_repeat_ngram_size` can be used by all models, I think  it'd be better to instead pass it to the init to `PretrainedConfig` under `decoder_start_token_id=decoder_start_token_id`

Reply:
Done !